### PR TITLE
Add more explicit path definitions to pontoon Fluent config

### PR DESCRIPTION
**Pull request by alexgibson**
See discussion in https://github.com/mozilla-l10n/www-l10n/pull/146

@pmac r? It seems like these config files don't quite work the way we originally thought, and we can't use broad wild cards like the way we have been if we need to expose certain files only to a subset of locales. 

This fix is a little verbose, but (from what I can tell) it's the only real option we have that doesn't involve re-architecting all the L10n files into different directories (which feels like a somewhat large change to make suddenly).

Thoughts?

### CODE REVIEW

File: l10n/configs/pontoon.toml [hunk 1]
```diff
@@ -115,6 +115,55 @@ locales = [
...
+    reference = "en/firefox/set-as-default/**/*.ftl"
+    l10n = "{locale}/firefox/set-as-default/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/welcome/**/*.ftl"
+    l10n = "{locale}/firefox/welcome/**/*.ftl"
+[[paths]]
+    reference = "en/firefox/whatsnew/**/*.ftl"
+    l10n = "{locale}/firefox/whatsnew/**/*.ftl"
+[[paths]]
+    reference = "en/mozorg/about/**/*.ftl"
```
Comment (pmac) [inline l10n/configs/pontoon.toml:159]: Doesn't this one include the next two, or is it really only those within subfolders of `about`?

**Review by pmac — APPROVED** [review]
Well this is unfortunate, but it is what it is. Looks good minus my one question below. 🥦

File: l10n/configs/pontoon.toml (hunk 1 above)
Comment (alexgibson) [reply l10n/configs/pontoon.toml:159]: Oh, you're right - shows how brittle this can be :/

**Comment by alexgibson** [comment]
@pmac pushed a fix, thanks

**Pull request merged**